cmd/tarmak/cmd/cluster.go

```diff
@@ -49,6 +57,32 @@ func clusterDestroyFlags(fs *flag.FlagSet) {
 	)
 }
 
+func clusterLogsFlags(fs *flag.FlagSet) {
+	store := &globalFlags.Cluster.Logs
+	now := time.Now()
+
+	fs.StringVar(
+		&store.Path,
+		"path",
+		utils.DefaultLogsPathPlaceholder,
+		"target tar ball path",
+	)
```

Comment:
Maybe we default to the current working directory, config files for tarmak should not be polluted

Reply:
yeah I also think current working directory is a better solution.

Reply:
:+1: